doc/devel/coding_style_guideline.rst

```diff
@@ -241,7 +241,7 @@ following guidelines:
   parameters, e.g. b-values, b-vectors, timings, and others.*
 * Cite the relevant papers. Use the *[NameYear]* convention for
   cross-referencing them, such as in [Garyfallidis2014]_, and put them
-  under the :ref:`references` section.
+  under the :ref:``references`` section.
```

Comment:
@areeshatariq This is an internal cross-reference: simple backticks must be used:
https://www.sphinx-doc.org/en/1.5/markup/inline.html

Reply:
Addressed in ac80972.